handlers/delete.go

```diff
@@ -97,7 +97,7 @@ func deleteFunction(functionNamespace string, clientset *kubernetes.Clientset, r
 	foregroundPolicy := metav1.DeletePropagationForeground
 	opts := &metav1.DeleteOptions{PropagationPolicy: &foregroundPolicy}
 
-	if deployErr := clientset.Apps().Deployments(functionNamespace).
+	if deployErr := clientset.AppsV1().Deployments(functionNamespace).
```

Comment:
Main breaking change that was made by the authors upstream was to add V1 to Apps/Core